wasi/test.c

```diff
@@ -39,10 +39,11 @@ testResolvePath(
     fprintf(stderr, "OK: resolvePath(%s, %s) == %s\n", directory, path, expected);
 }
 
+wasmMemory wasiMemory;
+
 int
 main(int argc, char* argv[]) {
-    wasmMemory m;
```

Comment:
Any particular reason for making this global?

Reply:
Ah, I see. `wasiMemory` is expected to be a function that the WASI implementation can call. 
Thanks for noticing this! 

See the suggestion above